### ISSUE TITLE
Implement CEL and GMPCA metrics

### ISSUE THREAD

**Issue by JoseAngelMartinB** [JoseAngelMartinB/gklr#7]
Currently, GKLR uses the log-likelihood to estimate the model.
It is purposed to implement the following modifications:

- Implement the Cross-Entropy Loss (CEL) and Geometric Mean Probability of Correct Assignment (GMPCA) metrics.
- Report those metrics on the model .summary() method.
- Modify the loss function of the model. Currently, the Log-likelihood function is used. It is purposed to use the CEL.
